src/services/roster/index.js

```diff
@@ -111,10 +130,16 @@ module.exports = function roster() {
 			],
 		},
 	};
+	const userGroupRoute = '/roster/users/:user/groups';
+	app.use(userGroupRoute, userGroupsHandler);
+	app.service(userGroupRoute).hooks(userGroupsHooks);
 
-	app.use('/roster/users/:user/groups', userGroupsHandler);
-	app.service('/roster/users/:user/groups').hooks(userGroupsHooks);
-
+	/**
+	 * Takes a course id and returns the pseudonyms of the group members
+	 * @param id ID of the given course
+	 * @returns {Array} data.students student ids and pseudonyms of students which are enrolled in the course
+	 * @returns {Array} data.teacers teacher ids and pseudonyms of teachers which are enrolled in the course
+	 */
```

Comment:
groups handler must require courseId and at least one toolId to be given to not return just one course from all courses. input validation for id, params.toolIds